projects/core/directives/dropdown/dropdown-hover.options.ts

```diff
@@ -15,8 +16,10 @@ export const TUI_DROPDOWN_HOVER_DEFAULT_OPTIONS: TuiDropdownHoverOptions = {
 /**
  * Default parameters for dropdown hover directive
  */
-export const TUI_DROPDOWN_HOVER_OPTIONS = tuiCreateToken(
-    TUI_DROPDOWN_HOVER_DEFAULT_OPTIONS,
+export const TUI_DROPDOWN_HOVER_OPTIONS = tuiCreateTokenFromFactory(() =>
+    inject(TUI_REDUCED_MOTION)
+        ? {showDelay: 0, hideDelay: 0}
```

Comment:
That is not an animation, that is a delay. Let's roll it back.

Reply:
fixed